lib/bundler/injector.rb

```diff
@@ -84,9 +84,6 @@ def build_gem_lines(conservative_versioning)
     def append_to(gemfile_path, new_gem_lines)
       gemfile_path.open("a") do |f|
         f.puts
-        if @options["timestamp"] || @options["timestamp"].nil?
```

Comment:
we should probably give a deprecation warning is the `timestamp` option is given via the command line though?